integration/pubsub_test.go

```diff
@@ -114,6 +114,33 @@ func (suite *serviceBusSuite) TestPublishAndListenRetryLater() {
 	}, event)
 }
 
+func (suite *serviceBusSuite) TestPublishAndListenShortLockDuration() {
+	suite.T().Parallel()
+	// creating a separate topic that was not created at the beginning of the test suite
+	// note that this topic will also be deleted at the tear down of the suite due to the tagID at the end of the topic name
+	shortLockTopic := suite.Prefix + "shortlock" + suite.TagID
+	pub, err := publisher.New(shortLockTopic, suite.publisherAuthOption)
+	suite.NoError(err)
+	l, err := listener.New(
+		suite.listenerAuthOption,
+		listener.WithSubscriptionDetails(2*time.Second, 3),
```

Comment:
Sigh my WithSubscriptionDetails isn't very explicit about what these values are setting. My fault.

Reply:
yes, I say it derived a little from the pattern. not a big deal, but we can fix it in a future PR, and break ALL OUR USERS ;)